ext/sysvshm/sysvshm.stub.php

```diff
@@ -6,16 +6,16 @@ final class SysvSharedMemory
 {
 }
 
-function shm_attach(int $key, ?int $memsize = null, int $perm = 0666): SysvSharedMemory|false {}
+function shm_attach(int $key, ?int $memory_size = null, int $permissions = 0666): SysvSharedMemory|false {}
```

Comment:
```suggestion
function shm_attach(int $key, ?int $size = null, int $permissions = 0666): SysvSharedMemory|false {}
```
is sufficient here, I think. We used just `$size` in shmop.

Reply:
yes, good catch, and I agree!